.circleci/config.yml

```diff
@@ -349,7 +349,6 @@ jobs:
       - restore_cache:
           keys:
             - bundler-dependencies-{{ checksum "Gemfile.lock" }}
-      - run: export CODECOV_FLAG=verifications
```

Comment:
Why did you remove this?

Reply:
Oops, sorry. That should've been part of a separate. Those are legacy environment variables we no longer use, but their removal is completely unrelated to the other change.